images/runtime/node/18/bullseye.Dockerfile

```diff
@@ -13,11 +13,11 @@ ENV GIT_COMMIT=${GIT_COMMIT}
 ENV BUILD_NUMBER=${BUILD_NUMBER}
 RUN ./build.sh node /opt/startupcmdgen/startupcmdgen
 
-FROM mcr.microsoft.com/oryx/base:node-18-20221112.1
+FROM mcr.microsoft.com/oryx/base:node-18-debian-bullseye-20230707.2
```

Comment:
For my own understanding - I'm not sure why we have this file checked in only for node 18 and not 16 and 14.
Shouldn't this file be generated while running the [generateDockerfiles](https://github.com/microsoft/Oryx/blob/main/images/runtime/node/generateDockerfiles.sh) script?

Reply:
That's true, this file _should_ be generated via `generateDockerfiles.sh` during CI -- this file pattern is actually covered by the `.gitignore`, so I'm assuming the file was uploaded before we updated the `.gitignore`, so I'll delete it since it shouldn't be maintained within the repository.